docs/models/fossil/design_integrated_ultra_supercritical_powerplant.rst

```diff
@@ -0,0 +1,161 @@
+================================== ================================================================
+
+Model Structure
+---------------
+
+For the construction of the charge and discharge GDP design models, the power plant model described in :ref:`Ultra-Supercritical Power Plant` is used as the base model to integrate the storage system. The charge cycle of TES uses a heat exchanger, a splitter, a cooler, and a pump, while the discharge cycle uses a heat exchanger, a splitter, and a turbine `ST`, where `ST` is used to produce additional power using the steam generated from the discharge heat exchanger.  Both, charge and discharge design models use unit models from the IDAES power generation unit model library, the IAPWS property package for steam and water, and the property packages for Solar Salt Property Package, Hitec Salt Property Package, and :ref:`Thermal Oil Property Package`. The unit models used in the design superstructures are shown in the table below:
+
+================================= =====================================================================
+Unit Model                        Units in Flowsheet
+================================= =====================================================================
+:math:`HelmTurbineStage`          Turbines (:math:`T_1` to :math:`T_{11}`) and :math:`BFPT`
+:math:`HelmSplitter`              Turbine Splitters
+:math:`Heater`                    Boiler Components (:math:`Boiler`, :math:`RH_1`, and :math:`RH_2`)
+:math:`HelmMixer`                 Mixers (:math:`CM` and :math:`DA`)
+:math:`HelmIsentropicCompresssor` Pumps (:math:`CP`, :math:`BP`, :math:`BFWP`, and :math:`SP`)
+:math:`HeatExchanger`             Condenser, Feed Water Heaters (:math:`FWH_1` to :math:`FWH_9`),
+                                  and Storage Systems (:math:`HXC` and :math:`HXD`)
+================================= =====================================================================
+
+Discrete Design Decisions
+-------------------------
+The charge superstructure includes two disjunctions for the optimal integration of the charge cycle. Disjunction 1 selects either Solar salt, Hitec salt, or Thermal oil as the heat transfer fluid for the charge heat exchanger. Disjunction 2 selects between a very high-pressure or high-pressure steam as the heat source for the charge heat exchanger. Given the two disjunctions, the charge superstructure considers 6 alternative flowsheet configurations. The different design alternatives are given in the following table:
+
+============================== ============ ========================================================
+Charge Design Decisions         Disjunction Description
+============================== ============ ========================================================
+:math:`solar_-salt_-disjunct`  1            Disjunct 1 to select solar salt as the storage material in charge heat exchanger
```

Comment:
Just curious. You mentioned that hyphen is needed to prevent `salt` from becoming a subscript. Would `solar\_salt\_disjunct` resolve the issue? This works in latex, but I'm not sure how rst files are rendered.